src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -191,6 +201,11 @@ void attachExceptionRecord(
                 Event.builder().summary(eventSummary).id(event.getEventId()).build()
             );
         } catch (FeignException e) {
+            log.error(
+                "Failed attaching exception record. Service response: {}",
+                e.contentUTF8(),
+                e
+            );
             throw new CcdCallException(
                 format("Internal Error: submitting attach file event failed case: %s Error: %s", caseRef, e.status()),
                 e
```

Comment:
It would be good to add the following TODO:
```
// TODO: remove the logging and move the message to the exception that's being thrown
// when it stops being returned to the caseworker
```

Current problem is that we'll log this information twice. Instead, we could just update the exception message, but we can't until caseworkers start receiving generic errors from our service, not ones containing details of what went wrong.